cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/store/DefaultStore.java

```diff
@@ -117,15 +117,12 @@
   private static final Gson GSON = new Gson();
   private static final Map<String, String> EMPTY_STRING_MAP = ImmutableMap.of();
   private static final Type STRING_MAP_TYPE = new TypeToken<Map<String, String>>() { }.getType();
-  private static final AtomicBoolean appMetaStoreInitialized = new AtomicBoolean(false);
-  private static final AtomicBoolean upgradeComplete = new AtomicBoolean(false);
-  private static final AtomicBoolean cacheLoaderInitialized = new AtomicBoolean(false);
 
   private final CConfiguration configuration;
   private final DatasetFramework dsFramework;
   private final Transactional transactional;
-
-  private static LoadingCache<byte[], Boolean> upgradeCacheLoader;
```

Comment:
It was simpler to just not use a static variable and a separate AtomicBoolean to initialize it.